pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+### Knowledge Base
+<!--
+_If the PR introduces new concept, then provide link(s) to the Knowledge base ( Documentation, Blog etc )_
+-->
+
+### Example(s)
+<!--
+_Any details that will help reviewers to review the PR, such as  (but not limited to ) expected request/response, instruction for viewing the changes via the web client, or any other relevant information._
+-->
+
+### Checklist
+
+- [ ] Black formatting
+- [ ] Tests
```

Comment:
Do you mean whether the tests ran successfully, or whether a PR contains new tests? The first should be an automated check.

Reply:
Thank you @azangru  for your valuable suggestions. I think this is just first attempt for us to have some formal structure of the PR. We can refine/udpate as we see fit. What I meant in the checklist is if people want to highlight something then this checklist would be handy. What goes there or removed from there that would depend on the task. I remember adding [ ] CI/CD in the ensembl-client PR whenever there is some change in CI/CD pipeline ( when you had checklist previously).